sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -43,27 +44,58 @@ have to be online at the same time.
 - [Maven][maven]
 - Microsoft Azure subscription
   - You can create a free account at: https://azure.microsoft.com
+- Azure Service Bus instance
+  - Step-by-step guide for [creating a Service Bus instance using Azure Portal][service_bus_create]
 
-### Adding the package to your product
+### Include the package
 
 [//]: # ({x-version-update-start;com.azure:azure-messaging-servicebus;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-messaging-servicebus</artifactId>
-    <version>7.0.0-beta.1</version>
+    <version>7.0.0-beta.2</version>
 </dependency>
 ```
 [//]: # ({x-version-update-end})
 
 ### Authenticate the client
 
-For the Service Bus client library to interact with an Service Bus, it will need to understand how to connect
-and authorize with it.
+For the Service Bus client library to interact with Service Bus, it will need to understand how to connect and authorize
+with it.
+
+#### Create Service Bus clients using a connection string
+
+The easiest means for doing so is to use a connection string, which automatically created when creating a Service Bus
```

Comment:
I know it is redundant when reading the README sequentially but we should say `The easiest means for authenticating`, using `doing so` out of context is confusing.

```suggestion
The easiest means for authenticating is to use a connection string, which automatically created when creating a Service Bus
```